README.md

```diff
@@ -482,6 +482,31 @@ parameters:
           RecordKey: "value"
 ```
 
+The output fields of the aggregates stage are:
+- `name`
+- `operation`
+- `record_key`
+- `by`
+- `aggregate`
+- `total_value`: the total aggregate value
+- `total_count`: the total count
+- `recent_raw_values`: a slice with the raw values of the recent batch
+- `recent_op_value`: the aggregate value of the recent batch
+- `recent_count`: the count of flowlogs in the recent batch
+
+These fields are used by the next stage (for example `prom` encoder).
+The pipeline processes flowlogs in batches.
+The output fields with `recent_` prefix are related to the recent batch.
+They are needed when exposing metrics in Prometheus using Counters and Histograms.
+Prometheus Counters API accepts the amount to be added to the counter and not the final value as in Gauges.
```

Comment:
- I suggest to add the word `delta`
- I suggest replacing `final value` with `total value`